src/main/java/com/example/petbuddybackend/controller/OfferController.java

```diff
@@ -44,6 +45,44 @@ public OfferDTO addOrEditOffer(@RequestBody @Valid ModifyOfferDTO offer,
         return offerService.addOrEditOffer(offer, principal.getName());
     }
 
+    @Operation(
+            summary = "Add configurations for offer",
+            description = "Adds configurations for offer. If configuration already exists, it will throw exception." +
+                    " If configuration does not exists, it will be added."
+    )
+    @PostMapping("/{offerId}/add-configurations")
```

Comment:
```suggestion
    @PostMapping("/{offerId}/configurations")
```

imo wiadomo że dodajemy konfigurację